firebase-functions/third_party/dagger2/LICENSE

```diff
@@ -4,7 +4,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
typo?

Reply:
It's 3 spaces everywhere... I fixed it in each lib's notice file